sqlglot/dialects/bigquery.py

```diff
@@ -346,6 +346,15 @@ def _json_extract_sql(self: BigQuery.Generator, expression: JSON_EXTRACT_TYPE) -
     return sql
 
 
+def _export_sql(self: BigQuery.Generator, expression: exp.Export) -> str:
+    return (
+        f"EXPORT DATA "
+        f"{f'WITH CONNECTION {self.sql(expression, 'with_connection')} ' if expression.args.get('with_connection') else ''}"
+        f"{f'{self.properties_sql(expression.args.get('options'))} ' if expression.args.get('options') else ''}"
+        f"{self.sql(expression, 'this')}"
+    )
```

Comment:
Let's move this to the base generator class and rename it into `export_sql`. That way it'll automatically be used when generating `exp.Export`.

Additionally, please refactor the implementation so that it's consistent with the rest of the generator's methods, i.e., don't do this inline in a single string. Use variables and leverage `self.sql`, etc.